extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -14,181 +14,141 @@ See the License for the specific language governing permissions and
+For semantics of configuration, please see [ad network documentation](https://developers.google.com/doubleclick-gpt/reference).
 
-`<meta name="amp-ad-refresh" content="network1=refresh_interval1,network2=refresh_interval2,...">`
 
-Where `refresh_interval` is the time, in seconds, in between refresh cycles. This value must be numeric and no less than 30. Individual slots may be opted-out of refresh by adding `data-enable-refresh=false` to the slot.
+#### Ad size
 
-#### Slot-level Configuration
+By default the ad size is based on the `width` and `height` attributes of the `amp-ad` tag. In order to explicitly request different ad dimensions from those values, pass the attributes `data-override-width` and `data-override-height` to the ad.
 
-An individual slot is eligible to be refreshed if it is configured as:
+Example:
 
+```html
+<amp-ad width=320 height=50
+    data-override-width=111
+    data-override-height=222
+    type="doubleclick"
+    data-slot="/4119129/mobile_ad_banner">
+</amp-ad>
 ```
-<amp-ad
- ...
- data-enable-refresh=refresh_interval>
-```
-If `refresh_interval` is set to false, then this slot will not be refresh-enabled, even if page-level configurations are set.
-
-#### SRA Compatibility
-
-Refresh is currently not supported for SRA enabled slots. If a slot is enabled for both, refresh will be disabled, and an error will be logged to the user's console.
 
-#### AMP Ad Container Compatibility
+For multi-size attributes, see the <a href="multi-size.md">multi-size documentation page</a>.
 
-The only AMP ad containers in which refresh is currently supported are amp-sticky-ad and amp-carousel container types.
+### Supported parameters
 
+- `data-slot`: Full path of the ad unit with the network code and unit code.
+- `data-multi-size`: See the <a href="multi-size.md">multi-size documentation page</a> for details.
+- `data-multi-size-validation`
 
-### SRA: Single Request Architecture (beta)
-Enabling SRA allows a publisher to make a single request for all ad slots on the AMP page which gives a publisher the ability to do roadblocking and competitive exclusions. This very similar to the behavior achieved on non-AMP pages when using [this](https://developers.google.com/doubleclick-gpt/reference#googletag.PubAdsService_enableSingleRequest) method in GPT.
+Supported via `json` attribute:
 
-In order to use this feature, add the following meta tag to the head of the AMP page:
-`<meta name=”amp-ad-doubleclick-sra”/>`
+- `categoryExclusions`: Sets a slot-level category exclusion for the given label name.
+- `cookieOptions`: Sets options for ignoring DFP cookies on the current page.
+  - 0: Enables DFP cookies on ad requests on the page. This option is set by default.
+  - 1: Ignores DFP cookies on subsequent ad requests and prevents cookies from being created on the page.
+- `tagForChildDirectedTreatment`: Configures whether the slot should be treated as child-directed.
```

Comment:
Could we say what the values here are?  Like, is the example below `=1` child-directed?  (Seeing rugby/cricket made me expect no, but normally `1` means yes.)

Reply:
Added links that [GPT developer documents](https://developers.google.com/doubleclick-gpt/reference#googletag.PassbackSlot_setTagForChildDirectedTreatment) reference.